tools/reproduce_comparison/Jenkinsfile

```diff
@@ -14,27 +14,34 @@ pipeline {
     stages {
         stage('Lets Compare') {
             agent {
-                label "${params.platform}"
-            }
+                label "${params.platform}&&build"
+            } 
             steps {
                 script {
-                    cleanWs()
                     try {
                         echo "Fetching artifact1 from ${params.URL1}"
-                        sh "mkdir url1Dir1 && wget -q ${params.URL1} -O - | tar -xz -C url1Dir1"
+                        def ret1 = sh returnStatus: true, script: "mkdir url1Dir1 && wget -q ${params.URL1} -O - | tar -xz -C url1Dir1"
```

Comment:
How would returnStatus ever get set to false?

Reply:
you are right! i need to fix this